src/config/firebase-config.js

```diff
@@ -0,0 +1,16 @@
+import firebase from "firebase";
+
+const firebaseConfig = {
+  apiKey: "AIzaSyAOJRaxF3xBfHAhcTFvCCqdhiID6T74Sk8",
+  authDomain: "little-tags-5ff8e.firebaseapp.com",
+  projectId: "little-tags-5ff8e",
+  storageBucket: "little-tags-5ff8e.appspot.com",
+  messagingSenderId: "828940268979",
+  appId: "1:828940268979:web:c64fe971acb414a9a247c6",
```

Comment:
It's a very bad idea to put secret in repos. Use ENV values of Netlify / Heroku.

Reply:
yes, removed and added .env file, [pr](https://github.com/pesto-students/little-tags-RenuSuresh-front-end/pull/13/files)

![image](https://user-images.githubusercontent.com/34194473/110129857-6ed3e100-7dee-11eb-94bd-eb625ac16ed8.png)